lib/floe/workflow/context.rb

```diff
@@ -147,6 +147,11 @@ def to_h
         safe_context
       end
 
+      def ==(other)
+        other.is_a?(self.class) && other.instance_variable_get("@context") == @context
+      end
+      alias eql? ==
```

Comment:
Pedantic, but if you override `.eql?` then that implies that `.hash` is the same, so you also need to override `.hash`.  Alternatively, I think if you override `.hash`, then `.eql?` works automatically and you don't need it.

Reply:
I did add `hash`, but thought that was too much.

Interestingly, implementing `hash` does not fix `eql?`